frontend/packages/ceph-storage-plugin/integration-tests/utils/consts.ts

```diff
@@ -33,16 +31,27 @@ export enum CLUSTER_STATUS {
 }
 
 export const OCS_NODE_LABEL = 'cluster.ocs.openshift.io/openshift-storage';
+export const CATALOG_SRC = 'redhat-operators';
 
 export const KIND = 'storagecluster';
 export const EXPAND_WAIT = 15 * MINUTE;
 export const CAPACITY_UNIT = 'TiB';
 export const CAPACITY_VALUE = '2';
-export const OCS_OPERATOR_NAME = 'ocs-operator';
+export const OCS_OPERATOR_NAME = 'ocs-operatorv4';
```

Comment:
What is this name? Haven't seen 'ocs-operatorv4' being used.

Reply:
For filtering purposes.